recipe/patches/0002-Consolidated-build-fixes-for-XLA.patch

```diff
@@ -399,6 +350,10 @@ index 0000000..6ff4e1d
 ++        "-labsl_log_internal_check_op",
 ++        "-labsl_log_internal_message",
 ++        "-labsl_log_internal_nullguard",
```

Comment:
Need `-labsl_log_internal_globals` to fix `undefined reference to `absl::lts_20240722::log_internal::IsInitialized()'`